client/landing/stepper/declarative-flow/registered-flows.ts

```diff
@@ -0,0 +1,65 @@
+import config from '@automattic/calypso-config';
+import type { FlowMetaData } from '../declarative-flow/internals/types';
+
+const availableFlows: Array< FlowMetaData > = [
```

Comment:
How about making it a mapping from flow name to download function or even a react.lazy component? Then,

* We don't need to search the flow from the array
* We can use `Suspense` and show the loading screen when the flow is downloading

What do you think?

Reply:
Oooh the mapping idea is brilliant! But we can't use React.lazy because the flows are not components, they're just objects.